src/features/search/components/Search.tsx

```diff
@@ -69,7 +60,7 @@ export const Search = ({
         </InputRightElement>
       </InputGroup>
       <Wrap w="100%" alignItems="start" pt={2}>
-        {availableTags.map((tag) => (
+        {tags?.map((tag) => (
```

Comment:
`tags?`?
why there is a question mark?

Reply:
because tags can be undefined when api response isn't there yet